src/data/case-studies/index.ts

```diff
@@ -1,27 +1,64 @@
 import { Group } from "@/types/interfaces";
 import output from "../../../public/data.json";
+import { technologies } from "./technologies";
+import { levels } from "./levels";
 
-const caseStudies = output.filter((data) => data?.PARSED_MANUAL_TAGS['CASE STUDY GROUP']);
-const toolKits = output.filter((data) => data?.PARSED_MANUAL_TAGS['CORE_TOOLKIT']);
+const caseStudies = output.filter(
+  (data) => data?.PARSED_MANUAL_TAGS["CASE STUDY GROUP"]
+);
+const toolKits = output.filter(
+  (data) => data?.PARSED_MANUAL_TAGS["CORE_TOOLKIT"]
+);
 
 const groups: Group[] = [
-    {
-        title: "",
-        blocks: [
-            {
-                title: "CORE Toolkits",
-                entries: toolKits
-            }
-        ]
-    },
-    {
-        title: "",
-        blocks: [
-            {
-                title: "Case Studies",
-                entries: caseStudies
-            }
-        ]
-    }
+  {
+    title: "",
+    blocks: [
+      {
+        title: "CORE Toolkits",
+        entries: toolKits,
+      },
+    ],
+  },
+  {
+    title: "",
+    blocks: [
+      {
+        title: "Case Studies",
+        entries: caseStudies,
+      },
+    ],
+  },
 ] as unknown as Group[];
-export default groups;
+
+const getEntryKeys = (highlightedElement: any) =>
```

Comment:
The data files should only contain the data and shouldn't contain these helper classes. please move these out of this file

Reply:
removed